README.rst

```diff
@@ -1,4 +1,7 @@
-.. image:: doc/source/logo/BBP-eModelRunner.jpg
+.. raw:: html
+
+    <img src="https://github.com/BlueBrain/EModelRunner/raw/main/doc/source/logo/BBP-eModelRunner.jpg"/>
+
```

Comment:
Could you put the link to the image at the end of the file, like the other images, with a substitution reference?

Also, could you use html for the other images in the readme (GUI_screenshot)?

Then you should also be able to delete the 'copy images referenced in Readme' part in doc/source/conf.py (lines 27 to 30).

Reply:
> Could you put the link to the image at the end of the file, like the other images, with a substitution reference?

Actually I don't know if html allows that.

> Then you should also be able to delete the 'copy images referenced in Readme' part in doc/source/conf.py (lines 27 to 30).

Oh that was the trick! I think copying the logo similar to the GUI_screenshot is even a better solution (with that it can get the image from the local repository, not from github's servers).